core/memory_tracker/cc/memory_tracker.h

```diff
@@ -320,9 +376,13 @@ class MemoryTracker {
   // By default |track_read| is set to false.
   MemoryTracker(bool track_read = false)
       : track_read_(track_read),
+#if IS_POSIX
```

Comment:
Should we use `if (TARGET_OS == GAPID_OS_LINUX) || (TARGET_OS == GAPID_OS_ANDROID)` to keep the thing consistent?